src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -308,19 +308,16 @@ class WelcomePage extends Disposable {
 				recent.classList.add('emptyRecent');
 				return;
 			}
-			const ul = container.querySelector('.recent ul');
+			const ul = container.querySelector('.recent ul') as HTMLElement;
 			if (!ul) {
 				return;
 			}
 			const workspacesToShow = workspaces.slice(0, 5);
-			const updateEntries = async () => {
-				while (ul.firstChild) {
-					ul.removeChild(ul.firstChild);
-				}
-				await this.mapListEntries(workspacesToShow, fileService, container);
+			const updateEntries = () => {
```

Comment:
You're no longer calling this onDidChangeFormatters - is that intentional? If you don't need to call this again then just get rid of the function and call these directly.

Reply:
fixed